tests/posit/connect/external/test_databricks.py

```diff
@@ -48,6 +48,7 @@ def test_posit_credentials_provider(self):
         register_mocks()
 
         client = Client(api_key="12345", url="https://connect.example/")
+        client.ctx.version = None
```

Comment:
We are setting the version to `None` to avoid mocking the API request to /server_settings.

Reply:
Should we put this in a helper function of some sort? Something like `test_client(...)` that does this internally. There are other ways too, but this is a lot of the same thing over and over (and a bit of specialized knowledge that you basically _always_ have to do this in these tests